chrome/content/zotero/xpcom/translation/translate_item.js

```diff
@@ -896,4 +899,4 @@ Zotero.Translate.ItemGetter.prototype = {
 		return obj;
 	}
 }
-Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
\ No newline at end of file
+Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
```

Comment:
It's in maintenance mode, but if there's a simple patch we can take it. (Not as big a deal when we can just skip the changes on merge conflict.)